### PR TITLE
Python example about Pagination

### PR DESCRIPTION
**Pull request by allmightyspiff**
Merged a bunch of listing articles into a single example about pagination

### CODE REVIEW

File: content/python/pagination.py.md [hunk 1]
```diff
@@ -1,52 +1,143 @@
...
-        offset = offset + limit
-        limit = limit + limit
-        print(result)
-        if not result:
-            break
-    except SoftLayer.SoftLayerAPIError as e:
-        print("Unable to retrieve the servers . " % (e.faultCode, e.faultString))
-    exit(1)
-    
+class Paginator():
```
Comment (caberos) [inline content/python/pagination.py.md:24]: the name is funny

File: content/python/pagination.py.md [hunk 2]
```diff
@@ -1,52 +1,143 @@
...
 ```
+
+## [A simple pagination example](#simple-example) {#simple-example .anchor-link}
+
+```python
+
+def list_servers(self):
+    orderBy = {"hardware": {"id": self.DESC}}
+    objectMask = "mask[id,hostname]"
+    servers = self.client.iter_call('SoftLayer_Account', 'getHardware', filter=orderBy, mask=objectMask, limit=10)
```
Comment (caberos) [inline content/python/pagination.py.md:51]: only 10 servers, is possible in the method parameter send this servers number to getting

File: content/python/pagination.py.md [hunk 3]
```diff
@@ -1,52 +1,143 @@
...
+    """This method will go through ALL invoices on an account.
+
+    The older an account is, the more invoices it will have, so queries like this must be split up. Otherwise
+    you will get API errors or timeouts. Once we have the invoice ID, we can get each invoices details seperately
+    which will greatly speed up this process
+    """
+    orderBy = {"invoices": {"id": self.DESC}}
+
+    objectMask = "mask[id, createDate, statusCode]"
+    invoices = self.client.iter_call("SoftLayer_Account", "getInvoices", filter=orderBy, mask=objectMask)
```
Comment (caberos) [inline content/python/pagination.py.md:109]: add a limit on method, this can produce a time out error in case the user has a much invoices

**Review by caberos — COMMENTED** [review]
some comments

File: content/python/pagination.py.md (hunk 3 above)
Comment (allmightyspiff) [reply content/python/pagination.py.md:109]: `iter_call` automatically adds a limit, the default is 100

File: content/python/pagination.py.md (hunk 2 above)
Comment (allmightyspiff) [reply content/python/pagination.py.md:51]: yes, but I'll leave that as an exercise for the reader.

File: content/python/pagination.py.md (hunk 1 above)
Comment (allmightyspiff) [reply content/python/pagination.py.md:24]: hopefully not funny in a bad way lol.  After looking this up, it seems `Paginator` might not be an actual real word (at least its not in [the dictionary](https://www.merriam-webster.com/dictionary/paginator) ), but it has some common uses for "something that does pagination".

**Review by FernandoOjeda — APPROVED** [review]
Looks good.

**Comment by caberos** [comment]
LGTM

**Pull request merged**